src/components/periodic-table/PeriodicTable.test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import PeriodicTable, { IPeriodicTableElement } from "./PeriodicTable";
+import {
+  render,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import PtElementInfo from "../pt-element/PtElementInfo";
+import ElementManager from "../../ElementManager";
+import cases from "jest-in-case";
+
+function elementRenderer(atomic: number): IPeriodicTableElement {
+  return {
+    // @ts-ignore fix this
```

Comment:
Any ideas on fixing this?

Reply:
I just looked at the original code and copied it
https://github.com/OpenSourceRaidGuild/atom-pwa/blob/next/src/components/periodic-table-page/PeriodicTablePage.tsx#L70